### PR TITLE
Link to Permission tab of Application Settings

### PR DESCRIPTION
**Pull request by meisenzahl**
Related to https://github.com/elementary/sideload/issues/97

### CODE REVIEW

**Comment by danrabbit** [comment]
Thanks for working on this! I'm looking at the pre-existing work we have in the GeoClue agent for displaying a link to system settings. It looks like the pattern there is to include a settings link in the body copy like this:

Screenshot from 2020-08-24 10 07 51@2x

Otherwise, if this button doesn't close the dialog it should be in the incidental action position like this:

Screenshot from 2020-08-24 10 11 43@2x

Or if it does close the dialog, it should be in the secondary action position like this:

Screenshot from 2020-08-24 10 12 21@2x

@cassidyjames any thoughts on which feels most natural?

**Comment by cassidyjames** [comment]
I'm not sure we need to link to permissions settings in the first place, as I believe the plan is to expose/explain what permissions are being requested (and the implications) before installing. But if we do want to show it here, then I think the inline link feels the best to me.

**Comment by meisenzahl** [comment]
Thanks for your feedback!

I have implemented the variant with the inline link.
If you like, you are welcome to use it :)

Screenshot von 2020-08-25 18 07 45

File: src/Views/SuccessView.vala [hunk 1]
```diff
@@ -43,15 +43,19 @@ public class Sideload.SuccessView : AbstractView {
                 primary_label.label = _("The app was installed successfully");
             }
 
-            secondary_label.label = _("Open it any time from the Applications Menu. Visit %s for app information, updates, and to uninstall.").printf (appstore_name);
+            secondary_label.label = _("Open it any time from the Applications Menu. Visit %s for app information, updates, and to uninstall. Permissions can be changed in <a href='%s'>%s → %s…</a>").printf (
+                appstore_name, "settings://applications/permissions", _("System Settings"), _("Applications")
```
Comment (danrabbit) [inline src/Views/SuccessView.vala:47]: Good call pulling out the URL so it doesn't get localized. I wonder if we need a translators comment for context on "Applications" or not?

Comment (meisenzahl) [reply src/Views/SuccessView.vala:47]: Something like this maybe?

`
/// TRANSLATORS: "System Settings" is related to the title of https://github.com/elementary/switchboard, "Applications" is related to the title of https://github.com/elementary/switchboard-plug-applications
`

Comment (danrabbit) [reply src/Views/SuccessView.vala:47]: Yeah that seems pretty comprehensive, let's do that

**Pull request merged**